src/notation/qml/MuseScore/NotationScene/internal/TextStylePopup.qml

```diff
@@ -0,0 +1,298 @@
+
+                    onValueEditingFinished: function(newValue) {
+                        //TODO
+                    }
+                }
+                RadioButtonGroup {
+                    id: verticalAlignmentButtonList
+
+                    height: 30
+                    width: implicitWidth
+
+                    model: [
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_TOP,
+                            typeRole: TextTypes.FONT_ALIGN_V_TOP,
+                            title: qsTrc("inspector", "Align top"),
+                            description: qsTrc("inspector", "Align top edge of text to reference point")
+                        },
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_MIDDLE,
+                            typeRole: TextTypes.FONT_ALIGN_V_CENTER,
+                            title: qsTrc("inspector", "Align middle"),
+                            description: qsTrc("inspector", "Align vertical center of text to reference point")
+                        },
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_BOTTOM,
+                            typeRole: TextTypes.FONT_ALIGN_V_BOTTOM,
+                            title:qsTrc("inspector", "Align bottom"),
+                            description: qsTrc("inspector", "Align bottom edge of text to reference point")
+                        },
+                        {
+                            iconRole: IconCode.TEXT_ALIGN_BASELINE,
+                            typeRole: TextTypes.FONT_ALIGN_V_BASELINE,
+                            title: qsTrc("inspector", "Align baseline"),
+                            description: qsTrc("inspector", "Align baseline of text to reference point")
+                        }
+                    ]
+
+                    delegate: FlatRadioButton {
+                        toolTipTitle: modelData.title
+                        toolTipDescription: modelData.description
+
+                        width: 30
+                        transparent: true
+
+                        iconCode: modelData.iconRole
+                        checked: false //currentVerticalAlignment(/*TODO*/)
+                        onToggled: {
+                            //TODO
+                        }
+                    }
+                }
+            }
+            RowLayout { //Column 2, Row 2
+                RadioButtonGroup {
+                    id: subscriptOptionsButtonList
+                    height: 30
+
+                    model: [
+                        { iconRole: IconCode.TEXT_SUBSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_BOTTOM, titleRole: qsTrc("inspector", "Subscript") },
+                        { iconRole: IconCode.TEXT_SUPERSCRIPT, typeRole: TextTypes.TEXT_SUBSCRIPT_TOP, titleRole: qsTrc("inspector", "Superscript") }
+                    ]
+
+                    delegate: FlatRadioButton {
+
```

Comment:
Code style: here is another extra empty line.

On the other hand, we do prefer to add an empty line between each two components, i.e.
```qml
    Whatever1 {
        …
    }
    // add empty line here
    Whatever2 {
        …
    }
```

To summarise: the following rules cover most cases:
- never an empty line after a line ending with `{`
- never an empty line between two `}` `}`
- almost always an empty line between `}` and a new component, property or function
- exception: in a list of JS objects, like for the `model` property of a `RadioButtonGroup`, we don't add a newline between two objects, i.e.
    ```qml
    model: [
        {
             key: value1
        },
        {
             key: value2
        }
    ]
    ```
    is correct.

Reply:
What about nested components like here?

`    RowLayout {
        id: menuItems

        ColumnLayout {
            StyledDropdown {`

I added a new line between the RowLayout and ColumnLayout because the RowLayout had extra properties and I had no new line between the ColumnLayout and the StyledDropdown